src/core/terminology.md

```diff
@@ -128,6 +128,22 @@ There are three coordinate systems for each camera:
     - example:
         - `(0, 0, 1)` (a point on the optical axis)
     
+- **Eye Model**:
+    - shares the 3D Camera Space coordinate system
+    - x: horizontal, y: vertical, z: optical axis
+    - examples:
+        - looking directly into the eye camera:
+            - `(x=0, y=0, z=-1)` (cartesian) 
+            - `phi=-π/2, theta=π/2` (spherical)
+        - looking up:
+            - decreasing `y` vector component; increasing `theta` 
+        - looking left:
+            - increasing `x` vector component; increasing `phi`
+        - see the cartesian to spherical coordinate calculation in our [sourcecode](https://github.com/pupil-labs/pupil/blob/eb8c2324f3fd558858ce33f3816972d93e02fcc6/pupil_src/shared_modules/pupil_detector_plugins/visualizer_pye3d/utilities.py#L14)
```

Comment:
I am not sure if this is a british vs american English thing, but I have never seen `sourcecode` as one word before. Feel free to dismiss this comment if the spelling is correct.
```suggestion
        - see the cartesian to spherical coordinate calculation in our [source code](https://github.com/pupil-labs/pupil/blob/eb8c2324f3fd558858ce33f3816972d93e02fcc6/pupil_src/shared_modules/pupil_detector_plugins/visualizer_pye3d/utilities.py#L14)
```